libvast/test/column_index.cpp

```diff
@@ -85,8 +85,8 @@ TEST(bro conn log) {
   for (auto slice : const_bro_conn_log_slices)
     col->add(slice);
   MESSAGE("verify column index");
-  auto pred = unbox(to<predicate>(":addr == 169.254.225.22"));
-  auto expected_result = make_ids({680, 682, 719, 720}, bro_conn_log.size());
+  auto pred = unbox(to<predicate>(":addr == 192.168.1.103"));
+  auto expected_result = make_ids({1, 3, 7, 14, 16}, bro_conn_log.size());
```

Comment:
My main question for this PR would be: how did you check the ground truth? Did you go back to the respective log or did you apply a "code transformation", e.g., to map 680 -> 1, 682 -> 3, etc.

Either is fine, but as the reviewer, I need to be convinced that it is correct.

Reply:
> [..] I gathered the expected values by taking the line numbers - 1 from my editor.

Sounds like a reasonable approach. 👍